src/pg/sql/helper.js

```diff
@@ -0,0 +1,47 @@
+import sql, { raw, join } from 'sql-template-tag';
+
+export const nowTimestamp = sql`cast(extract(epoch from now()) as integer)`;
+
+export const getJsonBuildObject = (variadic) => {
+  const args = join(
+    Object.entries(variadic).map(([name, value]) => {
+      return sql`'${raw(name)}', ${value}`;
+    }),
+  );
+  return sql`jsonb_build_object(${args})`;
+};
+
+export const getSelectCols = (table) => {
+  // TODO: When we have a query object, get only the requested columns.
+  return sql`to_jsonb("${raw(table)}")`;
+};
+
+export const colsAndValues = (data, versionCol) => ({
```

Comment:
For consistency with other functions here which all start with `get` and return a PG SQL object, I suggest splitting this into two functions: `getInsertCols` and `getInsertVals`.